phpBB/phpbb/filesystem.php

```diff
@@ -1,50 +0,0 @@
-<?php
-/**
-*
-* This file is part of the phpBB Forum Software package.
-*
-* @copyright (c) phpBB Limited <https://www.phpbb.com>
-* @license GNU General Public License, version 2 (GPL-2.0)
-*
-* For full copyright and license information, please see
-* the docs/CREDITS.txt file.
-*
-*/
-
-namespace phpbb;
-
-/**
-* A class with various functions that are related to paths, files and the filesystem
-*/
-class filesystem
```

Comment:
please keep an empty and deprecated class extending \phpbb\filesystem\filesystem (to be removed: 3.3) for BC purpose